tests/xharness/AppRunner.cs

```diff
@@ -326,168 +326,226 @@ public async Task<ProcessExecutionResult> UninstallAsync ()
 			}
 		}
 
-		bool IsTouchUnitResult (XmlDocument log)
+		bool IsTouchUnitResult (StreamReader stream)
 		{
-			return log.SelectSingleNode ("/TouchUnitTestRun/NUnitOutput") != null;
+			// TouchUnitTestRun is the very first node in the TouchUnit xml result
+			// which is not preset in the xunit xml, therefore we know the runner
+			// wuite quickly
```

Comment:
minor typo: `quite`

Reply:
I should add 1$ in a bucket every time I have a typo :/